sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -189,7 +189,7 @@ export class Receiver {
 
     let receiverName;
     if (this._context.batchingReceiver) {
-      receiverName = BatchingReceiver.name;
+      receiverName = this._context.batchingReceiver.name;
```

Comment:
Yeah as updated on #2398, I had re-run the test yesterday soon as I noticed this. It still did not fail and I see the message getting completed. I believe this is because renewMessageLock() is invoked in context of streaming receiver and likely the batching receiver flag is undefined for instance holding the streaming receiver.